_rules/auto-update-text-efbfc7.md

```diff
@@ -36,7 +36,7 @@ This rule applies to any [HTML element][] that has a [visible][] [text node][] a
 
 ## Expectation
 
-For each test target there is at least one set of [instruments][instrument], where each [instrument][] is in the same [web page][] as the test target or can be found in a [clearly labeled location][] from that [web page][], to achieve at least one of the following objectives:
+For each test target, the text change is a [non essential text change][] and there is at least one set of [instruments][instrument], where each [instrument][] is in the same [web page][] as the test target or can be found in a [clearly labeled location][] from that [web page][], to achieve at least one of the following objectives:
```

Comment:
Coming back to this after way too long, I think this rule will actually be subject to the changes we are suggesting in the subjective applicability discussion. I think this would fall into the third category of "[Thing] is/has [subjective attribute]". In this case, filling in the blanks, it would be 'Text change is not non-essential'.

With this, we could move this into the applicability and not have to worry about having that test in the expectation which should really be there, and we wont need to include an example in the test cases.